ui/packages/catalog-ui-search/package.json

```diff
@@ -60,6 +60,7 @@
     "openlayers": "4.6.5",
     "plotly.js": "1.10.5",
     "q": "1.5.0",
+    "querystring": "^0.2.0",
```

Comment:
Is the ^ a typo?

Reply:
No, that's npm. It means that version or greater: https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004

Reply:
I did an `npm install querystring` and it added that line for me.

Reply:
I think some linter complains about using version ranges like that

Reply:
OK. I can remove it.